src/jit/importer.cpp

```diff
@@ -2878,24 +2878,36 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
                                        CORINFO_SIG_INFO *       sig,
                                        int                      memberRef,
                                        bool                     readonlyCall,
-                                       CorInfoIntrinsics *      pIntrinsicID)
+                                       CorInfoIntrinsics *      pIntrinsicID,
+                                       bool *                   pMustExpand)
```

Comment:
There is really no need to have this out param as i see it.  This routine itself could NO_WAY if it failed to expand the intrinsic.  But i see this method doesn't have a single exit point that could be the reason to add an out param and exposing this info to caller.

An option is to modify this routine to have a single exit point to avoid the need for pMustExpand.  I would leave it to you if you think it is a good idea.